README.md

```diff
@@ -719,7 +719,7 @@ Basic requirements:
 - No concerning bug is found when testing the wallet
 - Website supports HTTPS and 301 redirects HTTP requests
 - SSL certificate passes [Qualys SSL Labs SSL test](https://www.ssllabs.com/ssltest/)
-- Website serving executable code or requiring authentication uses HSTS with a max-age of at least 180 days
+- Website serving executable code or requiring authentication uses HSTS with a max-age of at least 180 days and is included in the [HSTS preload list](https://hstspreload.appspot.com/)
```

Comment:
@jameshilliard Hey, sorry that this PR hasn't been resolved just yet. Can you please move this to the Optional section as per the [comment](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1277#issuecomment-231561648) @crwatkins left?